server/etcdserver/metrics.go

```diff
@@ -164,6 +164,28 @@ var (
 		Name:      "limit",
 		Help:      "The file descriptor limit.",
 	})
+	v2applySec = prometheus.NewHistogramVec(prometheus.HistogramOpts{
+		Namespace: "etcd",
+		Subsystem: "server",
+		Name:      "v2_apply_duration_seconds",
```

Comment:
I'm sorry for late feedback: 

I wonder whether its worth to keep it as a separate metric as opposed to keeping "v2" "v3" as api_version label on a single metric. 

From analysis perspective, I think its easier to have a single chart / alert that does not care what was the method of delivery of the request.

Reply:
Yes, good question in fact. I was thinking about keeping two as they both use disjoint values for operation name (like v2 uses POST/GET vs v3 using Range/Txn) so mixing this together seemed to be quite messy to me.

Overall I don't have strong opinion about this, we can merge them together (likely adding new v2/v3 dimension to the metric) if this makes it easier to analyze.

Reply:
Most clients use one or the other, not mixing in practice. 
But I think that lines on the same chart  for (v2/POST) and (v3,Range) are fair to have all processed applies.

Reply:
OK, Don